src/io.rs

```diff
@@ -685,6 +717,39 @@ pub trait ReadBytesExt: io::Read {
         Ok(())
     }
 
+    /// Reads a sequence of signed 8 bit integers from the underlying reader.
+    ///
+    /// The given buffer is either filled completely or an error is returned.
+    /// If an error is returned, the contents of `dst` are unspecified.
+    ///
+    /// Note that since each `i8` is a single byte, no byte order conversions
+    /// are used. It is included for completeness.
```

Comment:
I don't think these docs are quite correct. In particular, it's not that `read_i8_into` is provided for completeness, but rather, that it allows the caller to read a bunch of unsigned bytes into a signed byte buffer without using `unsafe`. Could we explicitly state that? (This would also explain why we don't offer a `read_i8` method.)

Reply:
Okay, I've improved the docs here. What do you think?